ports/ffmpeg/1000-resolume-small-memory-allocations.patch

```diff
@@ -56,7 +56,7 @@ index 858633e8c7..b96eab0a7b 100644
  
 -    ret = av_buffer_create(data, size, av_buffer_default_free, NULL, 0);
 -    if (!ret)
--        av_freep(&data);
+         av_freep(&data);
```

Comment:
I think this is an error and should not be there.

Reply:
You're right! No idea how that got here. Fixed it.